rc/parinfer.kak

```diff
@@ -67,17 +107,17 @@ Modes:
         shift
         echo "select ${line}.${column},${line}.${column} $@"
     }
-}
+} catch %{
```

Comment:
What's the motivation here?  `fail` messages are already logged to the debug buffer, so this seems to prevent the command from throwing, while presenting to the user like it did.  That seems weird for composition of `parinfer` with other commands.

The comment in `parinfer-enable-window` indicates that it expects the `parinfer` command to actually fail.

Reply:
Oi!